katran/lib/bpf/balancer_consts.h

```diff
@@ -245,6 +248,10 @@
 #define INIT_JHASH_SEED_V6 MAX_VIPS
 #endif
 
+#ifndef LOCAL_DELIVERY_OPTIMIZATION
```

Comment:
this defines LOCAL_DELIVERY_OPTIMIZATION unconditionally. e.g. "if it is not defined - define it" . instead it should be just a comment that such define exists and user of the library could compile bpf code w/ it set (e.g. same as lines 256 - 277 where we do have a list of all possible defines which are not set by default